### PR TITLE
Add donate prompts

### PR DESCRIPTION
**Pull request by wrightmartin**
Update
----------

To do 
- [x] Whitespace fix
- [x] https://github.com/mysociety/internal/issues/11

Part of #309

Here's how the messages look now

screen shot 2016-06-01 at 17 24 05
screen shot 2016-06-01 at 17 30 15

A note for the future, the way the notices and messages are put together in Alaveteli is messy and quite hard to follow - I'll open a ticket to refactor these

### CODE REVIEW

**Comment by wrightmartin** [comment]
(forgot to note - this was using the 'static-states' branch of alaveteli core https://github.com/mysociety/alaveteli/tree/static-states)

**Comment by MyfanwyNixon** [comment]
Ah yes, I see your point @crowbot.
How about something like:

> Your donations help us make Freedom of Information more accessible, for everyone.

Bit tricky getting so many concepts into a short message! And it'd be quite tempting to take a Google Ads-style approach and rotate several variants until you see which one brings the best results..
